apps/web/src/pages/get-started/tabs/Setup.tsx

```diff
@@ -0,0 +1,367 @@
+import { NextJSLogo, SvelteLogo, H3Logo, RemixLogo, ExpressLogo, NuxtLogo } from '../Logos';
+import { motion } from 'framer-motion';
+import { useState } from 'react';
+import { Button, Code } from '@mantine/core';
+import { OnboardingStepsTimeline } from '../OnboardingSteps';
+import { CodeSnippet } from '../legacy-onboarding/components/CodeSnippet';
+import { useEnvironment } from '../../../hooks/useEnvironment';
+import { CodeEditor } from '../CodeBlock';
+import { TextElement } from '../TextElement';
+import { CardButton, CardButtonGroupWrapper } from './CardsButtonGroup';
+import { HStack } from '@novu/novui/jsx';
+import { css } from '@novu/novui/css';
+import { IconOutlineMenuBook } from '@novu/design-system';
+import { useSegment } from '../../../components/providers/SegmentProvider';
+
+export const buildGuides = [
+  {
+    id: 'nextjs',
+    title: 'Next.js',
+    logo: NextJSLogo,
+    steps: [
+      {
+        title: 'Initialize a new project',
+        content: () => {
+          // eslint-disable-next-line react-hooks/rules-of-hooks
+          const { environment } = useEnvironment();
+
+          return <CodeSnippet command={`npx novu@latest init --secret-key=${(environment as any)?.apiKeys[0].key}`} />;
+        },
+      },
+      {
+        title: 'Run the application server',
+        content: () => {
+          return <CodeSnippet command={`cd my-novu-app && npm run dev`} />;
+        },
+      },
```

Comment:
i wonder if it is better to merge this step into step 1 `Initialize a new project`